tests/test_range.py

```diff
@@ -36,7 +37,9 @@ def test_bound_helper():
     assert _Bound(1, inc=False, is_lower=False) < _Bound(1, inc=True, is_lower=True)
     assert _Bound(1, inc=False, is_lower=False) < _Bound(1, inc=True, is_lower=False)
 
-    assert _Bound(1, inc=False, is_lower=True) < _Bound(1, inc=True, is_lower=False)
+    # In this case first bound is a little greater than 1,
+    # the second is exactly 1, so A !< B
+    # assert _Bound(1, inc=False, is_lower=True) < _Bound(1, inc=True, is_lower=False)
```

Comment:
The comment is right. I think we should swap to the sides and keep the assert. No need to have this comment I think (though I appreciate that you put it ther for the review).

Reply:
Will you be able to correct it in the merge or should I change it?

Reply:
If you could just amend your commit that would be great.

Reply:
Updated in the new commit